arch/xtensa/src/esp32s2/Make.defs

```diff
@@ -79,8 +79,3 @@ ifeq ($(CONFIG_ESP32S2_UART),y)
 CMN_CSRCS += esp32s2_serial.c
 endif
 
-ifeq ($(CONFIG_ARCH_USE_MODULE_TEXT),y)
-CHIP_CSRCS += esp32s2_modtext.c
-CMN_ASRCS += xtensa_loadstore.S
-endif
```

Comment:
esp32s2 part is not obvious to me.
can you explain a bit?

Reply:
yeah... I'm about to delete it.. I just didn't want to spend time on it. This was a copy past from the ESP32 port that shouldn't even be here.

ESP32-S2 memory model is similar to ESP32-C3 not ESP32.  We don't have a separate region accessed only by I-Bus.  So its (now non-existent) text allocator would just allocate from the main heap and adjust the buffer to be accessed from I-Bus.  I didn't do this for ESP32-S2 because the port is new.  I want to wait until we have more into it.

Reply:
> I'm about to delete it..

This was in reference to the `esp32s2_user.c` part.

Reply:
ok. i got it. thank you.